apps/src/dance/ai/DanceAiModal.tsx

```diff
@@ -244,7 +233,7 @@ const DanceAiModal: React.FunctionComponent<DanceAiModalProps> = ({
     }
   }, [currentAiModalField, mode, calculateMinMax]);
 
-  const getLabels = () => {
+  const labels = useMemo(() => {
```

Comment:
The `getLabels` function was only called once. This is better written as a `useMemo` so we can calculate the value once and leave it alone w/o changing via re-render or a single function call later.

Also a good candidate to re-write into a separate hook in the future.

Reply:
@bencodeorg I remember when you initially added this, you had run into some weird side effects from using `useMemo`, right? It sounds like those dependency issues may have gotten already solved along the way, but just tagging you in case there was something else we're missing here.

Reply:
Ahh yes, was it that issue where the preview wouldn't show up blank intermittently if you re-opened the modal after generating an effect? That might have even been before we switched to live preview though...